src/core/entities/MRModelEntity.js

```diff
@@ -92,6 +92,14 @@ export class MRModelEntity extends MRDivEntity {
 
             this.loaded = true;
 
+            this.object3D.traverse(object => {
```

Comment:
if you're traversing over `this.object3D` make sure to use `MREntity`'s [`traverseObjects`](https://github.com/Volumetrics-io/mrjs/blob/main/src/core/MREntity.js#L542) instead of just `traverse`

it adds in an extra check for if it's the actual root object to reduce redunant checks

Reply:
I'm not sure that will have an impact here. the only children to object3D in this case will be that of the model, as sub models haven't been added yet.

Reply:
probably - just better convention to use ours instead of threejs internal for this - not a hardline blocker though

Reply:
that's a fair point, I'll make the switch now.